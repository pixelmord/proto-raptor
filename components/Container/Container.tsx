import { Container as ChakraContainer, ContainerProps as ChakraContainerProps } from '@chakra-ui/react';
import React, { PropsWithChildren } from 'react';

export type ContainerProps = ChakraContainerProps & {
  fullWidth?: boolean;
};
export const Container: React.FC<ContainerProps> = ({
  fullWidth = false,
  ...props
}: PropsWithChildren<ContainerProps>) => {
  const width = fullWidth ? '100%' : ['95%', '90%'];
  return <ChakraContainer width={width} {...props} />;
};
export default Container;
