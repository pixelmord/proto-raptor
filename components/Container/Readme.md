### Container (variant:'default')

```js
  <Container />
```
