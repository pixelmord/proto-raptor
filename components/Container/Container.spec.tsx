import React from 'react';
import { render } from 'test/utils';
import { Container } from './Container';

describe('The Container component', () => {
  it('should render', () => {
    const { asFragment } = render(<Container />);
    const html = asFragment();
    expect(html).toMatchSnapshot();
  });
});
