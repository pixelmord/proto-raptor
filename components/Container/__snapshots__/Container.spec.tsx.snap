// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Container component should render 1`] = `
<DocumentFragment>
  <div
    class="chakra-container css-m77idv"
  />
</DocumentFragment>
`;
